### PR TITLE
retrieved the data from json file

### PR DESCRIPTION
**Pull request by Rajkumar-V**
please review the code @mupputur , @amlya1230 , @harisri9948 , @Jsm-mani , @gopi023 , @chennakesava89 , @anilkumar-golla

### CODE REVIEW

**Review by Jsm-mani — APPROVED** [review]
nice improvement working fine

File: json_parsers/json_parser2.py [hunk 1]
```diff
@@ -1,36 +1,30 @@
...
-
-x=json.loads(data)
-
-print(x["mongodb"])
-
-print(type(x))
-
-
+class JSON: 
+
```
Comment (amlya1230) [inline json_parsers/json_parser2.py]: class name should not be a built-in module name and it should follow camel case
For example
JsonParser
JsonReader
You can choose the name as per your requirement

File: json_parsers/json_parser2.py [hunk 2]
```diff
@@ -1,36 +1,30 @@
...
-
-print(x["mongodb"])
-
-print(type(x))
-
-
+class JSON: 
+    
+    def json_read(self,path):
+
```
Comment (amlya1230) [inline json_parsers/json_parser2.py]: you can split this function 

def read_file(self, filepath):
      pass

def load_json_data(self, json_text):
      pass

File: json_parsers/json_parser2.py [hunk 3]
```diff
@@ -1,36 +1,30 @@
...
+        x=json.loads(data)
+    
+        return x
+
+if __name__ == "__main__":
+    
+    j=JSON()
+    
+    path1 = "D:\\python\\input.json"
```
Comment (amlya1230) [inline json_parsers/json_parser2.py]: already  mentioned path should be relative path from your project directory

File: json_parsers/json_parser2.py [hunk 4]
```diff
@@ -1,36 +1,30 @@
...
+    out_put1 = j.json_read(path1)
+    print ("id1:",out_put1["data"][0]["_id"])
+    print ("id2:",out_put1["data"][1]["_id"])
```
Comment (amlya1230) [inline json_parsers/json_parser2.py]: You gen define a function to get the ids

File: json_parsers/json_parser2.py [hunk 5]
```diff
@@ -1,36 +1,30 @@
...
+    
+    path2 = "D:\\python\\input2.json"
+    out_put2 = j.json_read(path2)
+    print (out_put2["mongodb"])
```
Comment (amlya1230) [inline json_parsers/json_parser2.py]: Please define a function to get the mongodb name

**Review by amlya1230 — CHANGES_REQUESTED** [review]
Please find the inline comments

**Pull request closed without merging**